src/main/java/io/jenkins/plugins/sprp/models/Step.java

```diff
@@ -0,0 +1,57 @@
+package io.jenkins.plugins.sprp.models;
+
+import java.util.AbstractMap;
+import java.util.HashMap;
+
+public class Step {
+    private String stepName;
+    private String defaultParameter;
+    private HashMap<String, Object> parameters;
+
+    Step(){
+
+    }
+
+    Step(String stepName){
+        System.out.println("stepname: " + stepName);
```

Comment:
This is probably a debug statement. Please add a `// TODO: remove this debug statement` above the line.